### ISSUE TITLE
How can I modify the ini that is used by the bundled PHP executable?

### ISSUE THREAD

**Issue by aschmelyun** [NativePHP/laravel#52]
Use case: I need to increase the max post size from the default 2M to something larger.

Is there a default directory that the bundled exec looks for an *.ini in? Going through the Application Support folder or the source for the electron/laravel packages didn't reveal any hints.

Thanks!

**Comment by simonhamp** [comment]
@aschmelyun we've got an open issue elsewhere to add this as a feature.

I'll bring some of that together soon so we can keep track, but suffice to say for now we're planning to put together a neat DX for you to define these kinds of ini options 😉

**Comment by aschmelyun** [comment]
@simonhamp Sweet, thanks for the quick reply! Feel free to close this issue then if you don't want a ton of duplicates popping up 😄

**Comment by simonhamp** [comment]
@aschmelyun This has been implemented in #98 and [docs are online](https://nativephp.com/docs/1/getting-started/configuration#customize-phpini) for it - just waiting for release

**Comment by ralphmorris** [comment]
Hey @mpociot 

Not sure if I'm being stupid but this isn't working for me. I've just installed NativePHP into a brand new Laravel app and added the following parameters to the phpIni method.

```
    public function phpIni(): array
    {
        return [
            'memory_limit' => '1024M',
            'upload_max_filesize' => '40M',
            'post_max_size' => '50M',
        ];
    }
```

I then restarted my app with `php artisan native:serve`, but if I put phpinfo() into my routes file none of my changes are showing and I'm still getting a PostTooLargeException when uploading a 10mb file. Am I missing a step?

Cheers!
